<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NurseWindow</class>
 <widget class="QDialog" name="NurseWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton_addResults">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>250</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Add test results</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_diabetes">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Diabetes:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_bloodType">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Blood type:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_rbc">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>RBC:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_wbc">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>WBC:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_cbc">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>200</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>CBC:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_diabetes">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>79</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_bloodType">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>110</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_rbc">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>140</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_wbc">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>170</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_cbc">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>200</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_patientNumber">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>51</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Phone Number: </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_patientName">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>20</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_patientNumber">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>50</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Patient Name: </string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
